policyengine_api/api.py

```diff
@@ -117,6 +118,10 @@
 
 app.route("/<country_id>/user_profile", methods=["PUT"])(update_user_profile)
 
+app.route("/<country_id>/tracer_analysis", methods=["GET"])(
+    trigger_tracer_analysis
```

Comment:
While this is a valid name, it doesn't follow standard naming conventions. Could you instead call this `get_tracer_analysis`?